connPFM/connectivity/ev.py

```diff
@@ -68,132 +40,128 @@ def event_detection(data_file, atlas, surrprefix="", sursufix="", nsur=100, segm
     [t, n] = z_ts.shape
 
     # calculate ets
-    ets, u, v = calculate_ets(z_ts, n)
+    ets, u, v = connectivity_utils.calculate_ets(z_ts, n)
+
+    # Initialize thresholded edge time-series matrix with zeros
+    etspeaks = np.zeros(ets.shape)
 
-    # calculate rss
-    rss = np.sqrt(np.sum(np.square(ets), axis=1))
-    # initialize array for null rss
+    # Initialize array of indices of detected peaks
+    idxpeak = np.zeros(t)
+
+    # Initialize RSS with zeros
+    rss = np.zeros(t)
```

Comment:
why is this declared here and not inside the if condition in line 72

Reply:
Because the function returns RSS regardless of the option provided.

Reply:
makes sense